ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js

```diff
@@ -76,11 +78,13 @@ export default class ConfirmApproveContent extends Component {
     symbol,
     title,
     showEdit,
+    showCustomizeGasModal = false,
```

Comment:
having trouble following why this is necessary?

Reply:
I think the naming here is super confusing because `showCustomizeGasModal` is a function in `ConfirmApproveContent` and you're using the same name for a boolean here in this method on the `ConfirmApproveContent` component.

Reply:
fixed